### PR TITLE
FreeBSD platform support.

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by kondrak** [comment]
Not being a FreeBSD user I'm curious about this - are there any additional requirements needed to build the game? (packages to install etc.) Vulkan has no official SDK support here, so I'm wondering what steps someone would have to take to get it to work (build steps in the README.md would be nice!).

File: README.md [hunk 1]
```diff
@@ -70,6 +70,11 @@ export VK_LAYER_PATH=$VULKAN_SDK/macOS/etc/vulkan/explicit_layer.d
 
 This project uses the Vulkan loader bundled with the SDK, rather than directly linking against `MoltenVK.framework`. This is done so that validation layers are available for debugging. Builds have been tested using MacOS 10.14.2.
 
+## FreeBSD
+- pkg install vulkan-tools vulkan-validation-layers
+- make sure your graphics drivers support Vulkan (run `vulkaninfo` to verify) - if not, you can get them with:
+- enter the `linux` directory and type `make release` or `make debug` depending on which variant you want to build - output binaries will be placed in `linux/releasex64` and `linux/debugx64` subdirectories respectively
+
```
[the remark this answers is not in the document]
Comment (devnexen) [reply README.md]: In fact those install other Vulkan dependencies implicitly as well (driver loader and so on).

[the remark this answers is not in the document]
Comment (kondrak) [reply README.md]: Alright, does the user still need to set the environment variables (like on Linux/MacOS) or is it handled automatically by running the above `pkg install`? I'm trying to think if the instruction is not missing anything relevant.

[the remark this answers is not in the document]
Comment (devnexen) [reply README.md]: Should not need the layer for example is in the right place e.g. `/usr/local/share/vulkan/explicit_layer.d`

**Pull request merged**